src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/18.diff.sql

```diff
@@ -0,0 +1,297 @@
+﻿/*************************************************************
+    Create partition function, scheme and clustered index to migrate data.
+**************************************************************/
```

Comment:
looks like 18.diff.sql and 18.sql just recently got merged into master with a recent PR a couple hours ago so you'll probably have to change this to 19.diff.sql and 19.sql

Reply:
Added new schema files for version 19.